src/Features/CSharp/Portable/RemoveUnusedParametersAndValues/CSharpRemoveUnusedParametersAndValuesDiagnosticAnalyzer.cs

```diff
@@ -32,6 +32,10 @@ protected override bool IsIfConditionalDirective(SyntaxNode node)
         protected override bool IsCallStatement(IExpressionStatementOperation expressionStatement)
             => false;
 
+        protected override bool IsExpressionOfExpressionBody(IExpressionStatementOperation expressionStatementOperation)
+            => expressionStatementOperation.Parent is IBlockOperation blockOperation &&
+               !blockOperation.Syntax.IsKind(SyntaxKind.Block);
```

Comment:
hrmmm... the `!blockOperation.Syntax.IsKind(SyntaxKind.Block)` seems a little iffy to me.  I'd much rather have it be an explicit `.IsKind(ArrowExpressionClause)` or something like that...